src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -497,7 +497,7 @@ TEST_F(TransportManagerImplTest, SendMessageToDevice) {
 #endif  // TELEMETRY_MONITOR
 
   EXPECT_EQ(E_SUCCESS, tm_.SendMessageToDevice(test_message_));
-  testing::Mock::AsyncVerifyAndClearExpectations(1000);
+  testing::Mock::AsyncVerifyAndClearExpectations(10000);
```

Comment:
@Kozoriz Use please const variable instead of magic number. Also please move it out to class member variables and use in all test cases.